green-eggs-app/components/AvatarMenu.jsx

```diff
@@ -1,17 +1,27 @@
 import { View, Text, StyleSheet } from 'react-native';
-import React from 'react';
+import React, { useContext } from 'react';
 import { Provider, Modal, Portal, Button } from 'react-native-paper';
 import { useNavigation } from '@react-navigation/native';
 import { signOut } from 'firebase/auth';
 import { auth } from '../config';
+import { EggsUserContext, useEggsUserContext } from './EggsUserProvider';
 import AddProfile from './AddProfile';
 
-export const handleLogout = () => {
```

Comment:
Just double-checking- the logout stuff has been moved into Appstack so that doesn't impact the functionality of logout, correct? Has this been tested?